snuba/datasets/events_processor_base.py

```diff
@@ -56,6 +56,7 @@ class InsertEvent(TypedDict):
     datetime: str  # snuba.settings.PAYLOAD_DATETIME_FORMAT
     data: MutableMapping[str, Any]
     primary_hash: str  # empty string represents None
+    hierarchical_hashes: Sequence[str]
```

Comment:
Why this? 
The processor takes the hierarchical_hashes field from the `data` MutableMapping above. 
I don't think anybody fills this in 
https://github.com/getsentry/snuba/blob/master/snuba/datasets/errors_processor.py#L82-L84

Reply:
Thank you, I was confused by my own code again here